src/docfx/lib/schema/JsonSchema.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class JsonSchema
+    {
+        public JsonSchemaType Type { get; set; }
```

Comment:
Required properties?
https://json-schema.org/understanding-json-schema/reference/object.html#required-properties

Reply:
Support it during SDP implementation